src/vm-snapshot-form/snapshot-form/snapshotForm.jsx

```diff
@@ -26,36 +25,36 @@ export const VmSnapshotForm = ({
           <Col xs={12}>
             {!hideName &&
               <Field
-                validate={nameRequired && required({ msg: errorMessages.name })}
+                validate={nameRequired && required({ msg: __('Required') })}
```

Comment:
does it makes sense to ensure the strings are similar to the rails ones? so we:
1. don't create new strings
2. consistent with the API and rails based UI ?

Reply:
Yes. It is created to match the manage-ui-classic. You can check #32 for mode info. Strings are taken directly from the previous form.